config/locales/resend_renewal_email.en.yml

```diff
@@ -2,9 +2,6 @@ en:
   resend_renewal_email:
     messages:
       success: Renewal email sent to %{email}
-      assisted:
-        heading: Sorry, there has been a problem re-sending the renewal email.
-        details: You have requested to resend a renewal email to the contact email address, but this address matches %{email}.
       error:
         heading: We could not send an email to %{email}
```

Comment:
Will this catch errors if the email is nil? So will it say "We could not send an email to (nil)"?

Reply:
I read the rest of the file and can see this isn't the case so ignore me!

Reply:
If email is nil, a `MissingContactEmailError` will be raised and `ResendRenewalEmailController` will catch it and display an error message.